app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,273 @@
+    <string name="about_author_string">%s en bijdragers.</string>
+    <string name="about_affiliation">In samenwerking met</string>
+    <string name="privacy_friendly">Deze app behoort tot de groep PRIVACYVRIENDELIJKE apps, ontwikkeld door de Technische Universiteit van Darmstadt. De broncode is vrijgegeven onder de GPLv3-licentie. Op de afbeeldingen berust auteursrecht van TU Darmstadt en copyright van Google Inc.</string>
+    <string name="more_info">Ga voor meer informatie naar onze</string>
+    <string name="about_libraries">Gebruikte bibliotheken:</string>
+    <string name="app_name_long">Privacyvriendelijke QR-scanner</string>
+    <string name="version_number">Versie v%s</string>
+    <string name="delete_items">Items verwijderen</string>
+
+    <string name="scan_text">Houd dit venster boven de code die je wilt uitlezen.</string>
+    <string name="action_clear">Geschiedenis wissen</string>
+    <string name="cannot_clear">De geschiedenis is al gewist</string>
+    <string name="knowRisks">Ik heb de link gecontroleerd en wil de website bezoeken.</string>
+    <string name="domain">Domeinnaam</string>
+    <string name="urlExplanation">QR-codes kunnen links bevatten die naar een nepsite leiden. Zo'n site kan je persoonlijke gegevens misbruiken (phishingwebsite) of malware (bijv. virussen) installeren op je apparaat.\n\n<b>Controleer</b> dus altijd of de link wel de link is die je verwachtte:</string>
+    <string name="further_info"><a href="https://www.secuso.informatik.tu-darmstadt.de/de/research/results/nophish-flyer-deutsch/unterstuetzung-bei-der-erkennung-von-phishing-angriffen/">Informatie</a> over het controleren van links (met dank aan de collega\'s van de TU Darmstadt). Als je de link niet vertrouwt, open hem dan niet en sla hem ook niet op.</string>
+    <string name="delete_history">Weet je zeker dat je geschiedenis wilt wissen?</string>
+    <string name="ok">Ja</string>
+    <string name="element_removed">Item verwijderd.</string>
+    <string name="conform_url">Geef aan of je de link gecontroleerd hebt.</string>
+    <string name="wifi_explanation">Deze versie van de app kan niet automatisch verbinding maken met Wi-Fi-netwerken omdat hiervoor een machtiging vereist is. Hieronder vind je de netwerkinformatie.</string>
+    <string name="encryption">Versleuteling</string>
+    <string name="action_about">Over</string>
+    <string name="history_disabled">Geschiedenis bijhouden is uitgeschakeld</string>
+    <string name="to">Geadresseerde:</string>
+    <string name="subject">Onderwerp:</string>
+
+    <string name="usability_hint2">Als je een code niet kunt scannen, vergroot dan de afstand tussen je apparaat en de code.</string>
+
+    <string name="description_contact">De gescande QR-code bevat contactinformatie.</string>
+    <string name="description_email">De gescande QR-code bevat een e-mailadres.</string>
+    <string name="description_send_email">De gescande QR-code bevat een ingevuld e-mailadres.</string>
```

Comment:
Translation differs from english meaning: Filled in E-Mail instead of E-Mail Adress

Reply:
Ah, I misunderstood that. Will change it.